serializers/src/glTF/2.0/babylon.glTFMaterial.ts

```diff
@@ -69,11 +74,25 @@ module BABYLON.GLTF2 {
                 diffuse: babylonStandardMaterial.diffuseColor,
                 opacity: babylonStandardMaterial.alpha,
                 specular: babylonStandardMaterial.specularColor || Color3.Black(),
-                glossiness: babylonStandardMaterial.specularPower / 256
+                glossiness: 1 - Math.pow(1 - babylonStandardMaterial.specularPower / (_GLTFMaterial.maxSpecularPower as number), 4)
             };
-            if (babylonStandardMaterial.specularTexture) {
 
+            let specularStrength = Math.max(babylonSpecularGlossiness.specular.r, Math.max(babylonSpecularGlossiness.specular.g, babylonSpecularGlossiness.specular.b));
+            const oneMinusDiffuse = 1 - Math.max(babylonSpecularGlossiness.diffuse.r, Math.max(babylonSpecularGlossiness.diffuse.g, babylonSpecularGlossiness.diffuse.b));
+
+            let alpha = 1;
+
+            if (specularStrength < _GLTFMaterial.epsilon) {
+                alpha = 0;
+            }
+            else {
+                alpha = Scalar.Clamp(oneMinusDiffuse / specularStrength);
             }
+            alpha = 1 - (1 - alpha)/(specularStrength + 1);
+
+            specularStrength = specularStrength * alpha * 0.5;

```

Comment:
As discussed offline, let's solidify the algorithm before putting in arbitrary scalars.